docs/noise/qcvv/parallel_xeb.ipynb

```diff
@@ -65,7 +65,7 @@
     "    import cirq\n",
```

Comment:
<p>Do you think that it's worth commenting that the default entangling gate in <code>parallel_two_qubit_xeb</code> is CZ even though this example uses sqrt_iswap? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
added